docs/developer-docs/smart-contracts/advanced-features/http-gateways.mdx

```diff
@@ -0,0 +1,105 @@
+---
+keywords: [advanced, tutorial, http, http gateways, gateways]
+---
+
+import { MarkdownChipRow } from "/src/components/Chip/MarkdownChipRow";
+
+# HTTP gateways
```

Comment:
This is meant to be a high level overview of HTTP gateways, and may include how to run a remote gateway in a future iteration, so the title should remain general rather than specific.